libqfieldsync/utils/qgis.py

```diff
@@ -19,8 +19,8 @@
  ***************************************************************************/
 """
 
-import logging
 import os
+import logging
 import tempfile
```

Comment:
Why is that? This should not happen, because the imports are sorted alphabetically, do you have `pre-commit` installed?

Reply:
Yes, I have it, and it did not suggest that to me, but from now on, I will take that into consideration. Thanks!